healthScore/tests.py

```diff
@@ -142,10 +143,10 @@ def setUp(self):
         )
 
         # Adding user data
-        u1 = User.objects.create(
+        u1 = User.objects.create_user(
             email="user1@example.com",
             name="User1",
-            password=make_password("userpass1"),
+            password="userpass1",
```

Comment:
The test cases are failing. Could you please check as to why?

Reply:
yes I found out it's because you and I implemented the unit tests for login view and registration view at the same time. Since you modified the views.py, that's why my test cases failed. I did some modifications and it passed now. Please check again!

Reply:
Ohh, alright!